Detectors/TPC/eventdisplay/include/TPCEventdisplay/HitDraw.h

```diff
@@ -0,0 +1,43 @@
+/********************************************************************************
+ *    Copyright (C) 2014 GSI Helmholtzzentrum fuer Schwerionenforschung GmbH    *
+ *                                                                              *
+ *              This software is distributed under the terms of the             *
+ *         GNU Lesser General Public Licence version 3 (LGPL) version 3,        *
+ *                  copied verbatim in the file "LICENSE"                       *
+ ********************************************************************************/
+/*
+ * FairMCPointsDraw.h
+ *
+ *  Created on: Apr 17, 2009
+ *      Author: stockman
+ */
+
+#ifndef FAIRMCPOINTDRAW_H_
+#define FAIRMCPOINTDRAW_H_
+
+#include "FairPointSetDraw.h"           // for FairPointSetDraw
+
+#include "Rtypes.h"                     // for HitDraw::Class, etc
+
+class TObject;
+class TVector3;
+
+namespace o2 {
+namespace TPC {
+
+class HitDraw: public FairPointSetDraw
```

Comment:
not crucial for the moment: But in principle I would find the following name "TPCHitDrawTask" a better name.
Then it is clear what we are drawing + that this thing is actually a FairTask.